e2e/cypress/integration/constants.ts

```diff
@@ -3,3 +3,4 @@ export const describeIfIAMV2 = iamVersion.match(/v2/) ? describe : describe.skip
 export const describeIfIAMV2p1 = iamVersion === 'v2.1' ? describe : describe.skip;
 export const runFlaky: boolean = Cypress.env('RUN_FLAKY') || false;
 export const itFlaky = runFlaky ? it : it.skip;
+export const adminToken: string = Cypress.env('ADMIN_TOKEN');
```

Comment:
Cypress will now depend on having the env variable `CYPRESS_ADMIN_TOKEN` always being set. This adds a little extra overhead when running these tests locally, but it'll help our API-only tests run much more quickly by avoiding the time-consuming browser login interaction